docs/fix-bricked-xteink.md

```
@@ -0,0 +1,106 @@
+
+![proceed at your own risk](./images/spiflash/fix_bricked.jpg)
+
+In this document i will explain how to install CrossPoint on a bricked device or a device stuck on a firmware with no way to install a new one.
+I will use a SPIFlash programmer to write the firmware directly to the memory chip in the Xteink, bypassing the ESP32-C3 chip.
+If your Xteink is not USB locked, you can always flash safely over USB. the following method is a **last resource**.
+
+- Here is my Xteink stuck on biscuit firmware.
+
+![biscuit](./images/spiflash/stuck_on_biscuit.jpg)
+
+### Tools
```

markdownlint-cli2 (0.23.0)

[warning] Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)